src/user/talks.rst

```diff
@@ -11,15 +11,18 @@ to learn more about conda-forge.
 Conferences
 -----------
 
-* **IOOS DMAC 2016**: `Better Python Packaging with Conda-Forge (slides only) <http://ocefpaf.github.io/talk_conda_packaging/>`_, by Filipe Fernandes
-* **SciPy 2016**: `Community-Powered Packaging with conda-forge <https://www.youtube.com/watch?v=Hacl_YFzZOw>`_, by Phillip Elson
-* **SciPyLA 2016**: `CondaForge and the Future of Scientific Python packaging (Portuguese) <https://www.youtube.com/watch?v=Y0Fv7zT4Jl0>`_ + `slides <http://ocefpaf.github.io/SciPyLA_2016_talk/>`_, by Filipe Fernandes
-* **PyData New York 2018**: `Conda Forge - Community Driven Packaging That Works for You <https://www.youtube.com/watch?v=qHdRPoPYiHE>`_, by Marius van Niekerk
-* **PyData Global 2021**: `conda-forge in 2021 <https://www.youtube.com/watch?v=N2XwK9BkJpA>`_, by Eric Dill
+* **IOOS DMAC 2016**: `Better Python Packaging with Conda-Forge (slides only) <http://ocefpaf.github.io/talk_conda_packaging/>`__, by Filipe Fernandes
+* **SciPy 2016**: `Community-Powered Packaging with conda-forge <https://www.youtube.com/watch?v=Hacl_YFzZOw>`__, by Phillip Elson
+* **SciPyLA 2016**: `CondaForge and the Future of Scientific Python packaging (Portuguese) <https://www.youtube.com/watch?v=Y0Fv7zT4Jl0>`__ + `slides <http://ocefpaf.github.io/SciPyLA_2016_talk/>`__, by Filipe Fernandes
```

Comment:
```suggestion
* **SciPyLA 2016**: `Conda-forge and the Future of Scientific Python packaging (Portuguese) <https://www.youtube.com/watch?v=Y0Fv7zT4Jl0>`__ + `slides <http://ocefpaf.github.io/SciPyLA_2016_talk/>`__, by Filipe Fernandes
```

Reply:
Oops, yes, that's the title of the video but the slides do use the hyphen 😬